### PR TITLE
Toggle option added

### PR DESCRIPTION
**Pull request by BobanBaby**
Toggle option added along with drag

### CODE REVIEW

**Comment by torbenkeller** [comment]
Thanks for the PR!

Before I merge it, could you please do this:
* reformat the code with a line length of 100 
* add an option in the constructor to enable the toggling

**Comment by BobanBaby** [comment]
Done !


Made some changes according to pedantic/analysis_options.yaml. Please do check those also

File: lib/src/raw_expandable_bottom_sheet.dart [hunk 1]
```diff
@@ -222,21 +228,25 @@ class ExpandableBottomSheetState extends State<ExpandableBottomSheet> with Ticke
...
-    double headerHeight = _headerKey.currentContext!.size!.height;
-    double footerHeight = _footerKey.currentContext!.size!.height;
-    double contentHeight = _contentKey.currentContext!.size!.height;
+    var headerHeight = _headerKey.currentContext!.size!.height;
+    var footerHeight = _footerKey.currentContext!.size!.height;
+    var contentHeight = _contentKey.currentContext!.size!.height;
 
-    double checkedPersistentContentHeight =
-        (widget.persistentContentHeight < contentHeight) ? widget.persistentContentHeight : contentHeight;
+    var checkedPersistentContentHeight = (widget.persistentContentHeight < contentHeight)
```
Comment (torbenkeller) [inline lib/src/raw_expandable_bottom_sheet.dart:243]: please keep the types

**Review by torbenkeller — CHANGES_REQUESTED** [review]

**Comment by BobanBaby** [comment]
Done Please check it

**Review by torbenkeller — APPROVED** [review]

**Comment by torbenkeller** [comment]
Thank you for contributing!

**Comment by torbenkeller** [comment]
The new version is now published to pub.dev

**Pull request merged**